### PR TITLE
Include metadata in release archive, update package names, reduce package size

### LINKED ISSUE #531: Fix native modules issue in Release builds for React+TS version
Dev mode - Everything works as expected end to end
Release build -
* Build process works without error:
Build steps:
  * Install dependencies: `npm install`
  * Run package script: `npm run package`
  * App is generated in `out/` in the format supported by platform it is built on

image

* Running the generated app works fine:
image

* Err occurs, after selecting a JSON file for import, i.e. when sqlite3 is being `required` by Sequelize.
Essentially, sqlite3 isn't being made available in the release build, since it is a native module, it needs some particular config changes in webpack so that it can be available in the release build too.
image

### LINKED ISSUE #621: Review dependencies and ensure we have the bare minimum dependencies declared & used at runtime

### LINKED ISSUE #622: Improve released archive content
### Description
- [ ] Include version in the name of the extracted release folder
- [x] Ensure there aren't any spaces in the archive or extracted folders
- [ ] Ensure that we have top-level files like Readme, License in the built archives
- [x] Add links for specific versions of chromium & electron used in the Release.md

### PR DESCRIPTION
**Pull request by OmkarPh**
Fixes #531 #621 #622 

- Removed story set illustrations (partially fixing #620)
- Added metadata files in the final package
- Include version & omit spaces in the package directory as well as archive
- Eliminated need to package entire node_modules, in order to support native modules as explained in #531, thanks to [@timfish/forge-externals-plugin](https://www.npmjs.com/package/@timfish/forge-externals-plugin)

You can see archives in this test CI release - https://github.com/OmkarPh/scancode-workbench/releases/tag/v4.0.1-src-packages

New archives are only ~20 MBs smaller, but the unpackaged application size has reduced from >400 MBs to ~300 MBs

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* Keep the scope as simple as possible to make it easier to implement.
+* Remember that your contributions are welcomed to implement this feature!
+
+
+Chat with other developers
+==========================
+
+For other questions, discussions, and chats, we have:
+
+- an official Gitter channel at https://gitter.im/aboutcode-org/discuss
```
Comment (AyanSinhaMahapatra) [inline CONTRIBUTING.rst]: Let's update this with https://matrix.to/#/#aboutcode-org_discuss:gitter.im and https://matrix.to/#/#aboutcode-org_scancode-workbench:gitter.im

File: workbench.ABOUT [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+homepage_url: https://www.aboutcode.org/
 license: apache-2.0
 owner: nexB Inc.
-copyright: Copyright (c) nexB Inc. and others. All rights reserved.
-
 notice_file: NOTICE
-license_file: LICENSE
+vcs_url: git+https://github.com/nexB/scancode-workbench
+bug_tracking_url: https://github.com/nexB/scancode-workbench/issues
 notes: ScanCode Workbench is licensed under Apache 2.0. Some example code from electron-quickstart is used and is licensed under CC0-1.0.
```
Comment (AyanSinhaMahapatra) [inline workbench.ABOUT]: is this `notes` still relevant and correct?

**Review by AyanSinhaMahapatra — APPROVED** [review]
@OmkarPh LGTM!
Thanks ++

See a couple minor comments below, and also update the relevant meta-issues

File: workbench.ABOUT (hunk 2 above)
Comment (OmkarPh) [reply workbench.ABOUT]: yep, thats outdated & not needed anymore

File: CONTRIBUTING.rst (hunk 1 above)
Comment (OmkarPh) [reply CONTRIBUTING.rst]: thanks for pointing this out

**Pull request merged**